File: Practice/21/C++/21/21.cpp
#include "pch.h"
#include <iostream>

// Принимает вес в килограммах и рост в метрах. Возвращает индекс массы тела. 
double BMI(double weight, double height);
// Принимает численное значение ИМТ и печатает на экран соответствующую категорию
void printBMI(double BMI);

int main()
{
	double weigt, height = 0;
	std::cin >> weigt >> height;
	printBMI(BMI(weigt,height / 100));
}

double BMI(double weight, double height) 
{
	return weight / (height * height);
}

void printBMI(double BMI) 
{
	if (BMI < 18.5) 
		std::cout << "Underweight";
	else if (BMI >= 18.5 && BMI < 25.0)
		std::cout << "Normal";	
	else if (BMI >=25.0 && BMI < 30.0)
		std::cout << "Overweight";
	else
		std::cout << "Obesity";
}

File: Practice/21/Python/21.py
def BMI(weight : float, height : float) -> float:
	return weight / (height * height)

def printBMI(BMI : float) -> float:
	if (BMI < 18.5):
		print("Underweight")
	elif (BMI >= 18.5 and BMI < 25.0):
		print("Normal")
	elif (BMI >=25.0 and BMI < 30.0):
		print("Overweight")
	else:
		print("Obesity")
		
weigt, height = map(float,input().split())
printBMI(BMI(weigt,height / 100))
